src/lib/utils.js

```diff
@@ -6,7 +6,7 @@ const {
 	readFileAsync,
 	writeFileAsync,
 	statAsync,
-	renameAsync,
+	renameAsync
```

Comment:
Trailing commas: https://github.com/balena-io/process/blob/master/process/TypeScript_Coding_Guide.md#trailing-commas
```suggestion
	renameAsync,
```